lib/utils.js

```diff
@@ -228,7 +229,11 @@ exports.getPantheonEnvironment = options => ({
   PATH: PATH.join(':'),
   PANTHEON_ENVIRONMENT: 'lando',
   PANTHEON_INDEX_HOST: PANTHEON_INDEX_HOST,
-  PANTHEON_INDEX_PORT: PANTHEON_INDEX_PORT,
+  PANTHEON_INDEX_PORT: options.solrTag === '8' ? '8983' : '449',
+  PANTHEON_INDEX_SCHEME: PANTHEON_INDEX_SCHEME,
+  PANTHEON_INDEX_PATH: '/',
```

Comment:
@reynoldsalec i am *pretty sure* these ENVARS will be SOLR version specific. So changing them wholesales runs the risk of breaking things for SOLR 3. 

It might make sense to:

* wrap them in some sort of "getSolrEnvars` function
* pull down an older pantheon site and test the older schema posting to see if it still works
* add a test for the older solr version